app/models/git/track.rb

```diff
@@ -69,6 +69,26 @@ def concepts
       config[:concepts].to_a
     end
 
+    memoize
+    def indent_style
+      config[:online_editor][:indent_style].to_sym
```

Comment:
1. We should probably have a default for both of these in case they're missing? `:space`?
2. I wonder if we should turn this into a boolean instead rather than comparing symbols further down the stack. So this would be `use_spaces_for_intent_style?`or something. Let's discuss this?
3. If we don't do (2), we should guard that it is either `:space` or `:tab`.

Reply:
This should never be missing due to configlet checking its presence. Still want to add a default?

Reply:
> I wonder if we should turn this into a boolean instead rather than comparing symbols further down the stack. So this would be use_spaces_for_intent_style?or something. Let's discuss this?

We could do that. I didn't do that as even though we use this to determine a boolean, I felt like that is something that is editor-specific, which the `Git::Track` class should not know about I feel.